app/models/core_induction_programme_exporter.rb

```diff
@@ -9,7 +9,7 @@ def run
       import: true,
     )
 
-    years = CourseYear.order(:created_at)
+    years = CourseYear.order(:title)
```

Comment:
Just wondering how this orders when titles can start with both lowercase and uppercase letters. Do we need to restrict case?

Reply:
I think titles of course years won't change any time soon.

The purpose of ordering in this file is to have a set order in migrations file. I thought `created_at` would more or less do that, but it was a little brittle, so I switched to something more stable and predictable.

As for 'do we need to worry about lower/upper case?' - very unlikely.